cterasdk/lib/utils.py

```diff
@@ -0,0 +1,14 @@
+import base64
+
+
+def merge(d1, d2):
```

Comment:
This method changes d1 in place. So, merge(a, b) + merge(a,c) will result in a+b+c. Is that the expected behavior?